SnapMD.VirtualCare.ApiModels/PatientUpdateRequest.cs

```diff
@@ -35,6 +35,6 @@ public class PatientUpdateRequest
         public int? OrganizationId { get; set; }
         public int? LocationId { get; set; }
         public IList<IdentifierValue> Identifiers { get; set; }
-        public string CountryCode { get; set; }
```

Comment:
This field is in use and might be used by API clients.  Probably it is not safe to rename without breaking client's code.  Please keep the old field (and mark it obsolete) when adding new ones.